samza-test/src/test/java/org/apache/samza/test/samzasql/SamzaSqlIntegrationTestHarness.java

```diff
@@ -45,6 +45,9 @@ protected void runApplication(Config config) {
     HashMap<String, String> mapConfig = new HashMap<>();
     mapConfig.put(JobConfig.JOB_COORDINATOR_SYSTEM, MOCK_METADATA_SYSTEM);
     mapConfig.put(String.format(SystemConfig.SYSTEM_FACTORY_FORMAT, MOCK_METADATA_SYSTEM), MockSystemFactory.class.getName());
+
+    // add some serde configs for the in-memory system
```

Comment:
can you update the comment to reflect that we setup job default system and system configs for in-memory system?